test-framework/sudo-compliance-tests/src/sudo/timestamp.rs

```diff
@@ -166,7 +166,7 @@ fn cached_credential_shared_with_target_user_that_is_self_on_the_same_tty() -> R
     Command::new("sh")
         .arg("-c")
         .arg(format!(
-            "echo {PASSWORD} | sudo -S true; sudo -u {USERNAME} env '{SUDO_RS_IS_UNSTABLE}' sudo -n true && true"
+            "echo {PASSWORD} | sudo -S true; sudo -u {USERNAME} sudo -n true && true"
```

Comment:
I think these all should remain in case you want to run the compliance tests with the dev feature enabled to debug an issue.

Reply:
I was considering that as well so good that you mention it. Basically that means we simply remove 9844f1d from this PR.